tests/lib/rules/no-unsupported-features/es-syntax.js

```diff
@@ -1112,55 +1308,60 @@ runTests([
                 ],
                 errors: [
                     {
-                        messageId: "no-property-shorthands",
-                        data: { supported: "4.0.0", version: "3.9.9" },
-                    },
-                ],
-            },
-            {
-                code: "({ get })",
-                options: [{ version: "3.9.9" }],
-                errors: [
-                    {
-                        messageId: "no-property-shorthands-getset",
-                        data: { supported: "6.0.0", version: "3.9.9" },
-                    },
-                ],
-            },
-            {
-                code: "({ set })",
-                options: [{ version: "3.9.9" }],
-                errors: [
-                    {
-                        messageId: "no-property-shorthands-getset",
-                        data: { supported: "6.0.0", version: "3.9.9" },
-                    },
-                ],
-            },
-            {
-                code: "({ get })",
-                options: [{ version: "5.9.9" }],
-                errors: [
-                    {
-                        messageId: "no-property-shorthands-getset",
-                        data: { supported: "6.0.0", version: "5.9.9" },
-                    },
-                ],
-            },
-            {
-                code: "({ set })",
-                options: [{ version: "5.9.9" }],
-                errors: [
-                    {
-                        messageId: "no-property-shorthands-getset",
-                        data: { supported: "6.0.0", version: "5.9.9" },
+                        messageId: "not-supported-till",
+                        data: {
+                            ruleId: "no-property-shorthands",
+                            supported: ">=4.0.0",
+                            version: "3.9.9",
+                        },
                     },
                 ],
             },
+            // TODO: Additional es-syntax
```

Comment:
I have commented out `no-property-shorthands-getset`, and `no-bigint-property-names`. as I want to cover them in a less custom way.

I am open to ideas/suggestions